clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -854,6 +854,14 @@ boolean reconcile() {
         return true;
     }
 
+    long getExpirationTimeForTimeout(final long timeoutMs) {
+        long expiration = System.currentTimeMillis() + timeoutMs;
```

Comment:
Can we please pass an instance of `Time` into this class like everywhere else? Debugging tests where half of the classes use a mock time and the other half uses wall-clock time will be quite confusing.

Reply:
Totally, agree it ends up being confusing in the tests. Done.